docs/usage/configuration-options.md

```diff
@@ -1556,6 +1556,18 @@ To adjust it down to 10s for all queries, do this:
 }
 ```
 
+### httpsCertificateAuthority
+
+`httpsCertificateAuthority` overrides any trusted CA certificate.
+
+### httpsPrivateKey
+
+Specifies the private key for mTLS authentication.
```

Comment:
It may be worth noting that this also needs to be in PEM format.

Reply:
added the PEM format as well as a note to tell user not to specify it directly